core/assertion.ts

```diff
@@ -1,17 +1,20 @@
-import { Resolvable, Session } from "@dataform/core/session";
+import { ICommonContext, ICommonOutputConfig, Resolvable } from "@dataform/core/common";
+import { Session } from "@dataform/core/session";
 import * as utils from "@dataform/core/utils";
 import { dataform } from "@dataform/protos";
 
-export type AContextable<T> = T | ((ctx: AssertionContext) => T);
+export interface IAssertionContext extends ICommonContext {}
 
-export interface AConfig {
-  dependencies?: Resolvable | Resolvable[];
-  tags?: string[];
-  description?: string;
-  database?: string;
-  schema?: string;
-}
+export interface IAssertionConfig extends ICommonOutputConfig {}
```

Comment:
can we override some of the documentation here? it's confusing that it refers to 'datasets' for assertion related code.